gst/droidcamsrc/gstdroidcamsrcdev.h

```diff
@@ -3,6 +3,8 @@
  *
  * Copyright (C) 2014 Mohammed Sameer <msameer@foolab.org>
  * Copyright (C) 2015-2016 Jolla LTD.
+ * Copyright (C) 2020 UBports foundation.
+ * Author: Ratchanan Srirattanamet <ratchanan@ubports.com>
```

Comment:
I noticed you took out the UBports credit out too - I was only talking about the Author ones. If your work is supported by UBports then it would be great to show this here.

Reply:
Oh, OK. Will add it back.